### DISCUSSION TITLE
still not getting full image of head to toe

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#3486] [Q&A]
hi all
trying for a full body image,  tried in the prompt standing , head to toe image, medium shot etc and not getting any full person images?

**Comment by Yogesh-DevHub** [comment]
Hi, 
There is a video on how to prompt specifically for Fooocus, this should help you. It has prompt for full body which works. 
https://youtu.be/VzCaIfFR8bs?si=qBqMN1m_V0AWn921

**Comment by lgwjames** [comment]
yep i know the basic, as been doing since feb. unless i add like wearing black shoes, etc doesn't want to do anything more than belly up for the last month

**Reply by poor7** [reply]
There is probably something wrong with your prompts or model. What model are you using for generation? Can you provide some examples of hints and the resulting images?

**Reply by lgwjames** [reply]
> There is probably something wrong with your prompts or model. What model are you using for generation? Can you provide some examples of hints and the resulting images?

model is juggernaultxlv8

**Reply by poor7** [reply]
> model is juggernaultxlv8

This model was inclined to close-ups. Try the model [Juggernaut-X](https://huggingface.co/RunDiffusion/Juggernaut-X-v10/resolve/main/Juggernaut-X-RunDiffusion-NSFW.safetensors)

**Reply by lgwjames** [reply]
> > model is juggernaultxlv8
> 
> This model was inclined to close-ups. Try the model [Juggernaut-X](https://huggingface.co/RunDiffusion/Juggernaut-X-v10/resolve/main/Juggernaut-X-RunDiffusion-NSFW.safetensors)

so no lora on (jessical) (set to 1:2) and same prompt, no jessica in it and get

Full body image, __ perspective__ , emotion, pose wearing a Clothing_Designers clothing-female , standing on beach in turkey, pencil eyeliner, DSLR, parted lips, pink lipstick, silver necklace with a cross on it, straw hat

![2024-08-13_16-58-17_7297](https://github.com/user-attachments/assets/bd725aaf-3c7d-4773-83b1-0e6b45ef420c)

changed the checkpoint as posted

i get 
![2024-08-13_17-01-42_1573](https://github.com/user-attachments/assets/c7143651-a221-4e80-83e9-3aef5d0b1cfb)


so changed back to jessical lora

added prompt

i wont post it, but basically chest up. and the second photo knees up

**Comment by lgwjames** [comment]
strange as some times I change the ratios of image size and it doesnt change the output

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by lgwjames** [comment]
prompt 
`__jessicaL__ , __perspective__ , __emotion__, pose wearing a __Clothing_Designers__ __clothing-female__ , standing on beach in turkey, pencil eyeliner, DSLR, parted lips, pink lipstick, silver necklace with a cross on it, straw hat, Ground level, (from the side) Full body image,`

set to 832x1216 and get

![2024-08-12_17-31-46_5763](https://github.com/user-attachments/assets/db164d1b-9156-4cda-bdf0-8206b083555c)

should be a longer image

changed to 

`__jessicaL__ , __perspective__ , __emotion__, pose wearing a __Clothing_Designers__ __clothing-female__ , standing on beach in turkey, pencil eyeliner, DSLR, parted lips, pink lipstick, silver necklace with a cross on it, straw hat, Full body image,`

set to 704x1408 ( should be a lot skinner pic )

![2024-08-12_17-35-01_4228](https://github.com/user-attachments/assets/eeedca37-a4b3-4b43-a4e4-488564379c90)

ideals?

**Comment by lgwjames** [comment]
changed model to realisticStockPhoto_v20
__jessicaL__ , __perspective__ , __emotion__, pose wearing a __Clothing_Designers__ __clothing-female__ , standing on beach in turkey, pencil eyeliner, DSLR, parted lips, pink lipstick, silver necklace with a cross on it, straw hat, Full body image,

![2024-08-12_17-39-16_7012](https://github.com/user-attachments/assets/96d870b7-2215-424c-aa13-9f392c592a2b)

**Comment by lgwjames** [comment]
so changed a few bits, preset realistic, size 704x1408,  got this ( looks more full body??)

![2024-08-12_17-53-50_4793](https://github.com/user-attachments/assets/744fbe4d-7519-41f1-b2ce-9fafa1feba61)
![2024-08-12_17-54-41_7701](https://github.com/user-attachments/assets/3edc5342-a130-40e9-a010-4089f917d9e8)
![2024-08-12_17-55-32_7625](https://github.com/user-attachments/assets/c3842ebb-7ce3-4dd8-977a-cb4d0efede50)
![2024-08-12_17-56-23_8468](https://github.com/user-attachments/assets/50feb4f9-b2be-4700-ae18-9eeff724cfb7)
![2024-08-12_17-57-51_6595](https://github.com/user-attachments/assets/ebf65a0a-9722-4790-bdf2-8b287a938309)
![2024-08-12_17-58-43_4142](https://github.com/user-attachments/assets/48f096b4-567c-480c-a9fc-9500390d5d15)
![2024-08-12_17-59-34_1860](https://github.com/user-attachments/assets/2da59c32-410c-4765-9dfa-4b29684f265c)

as soon as i add __jessicaL__ for my lora i go to prompt

__jessicaL__ ,__ perspective__ , __emotion__, __pose__ wearing a __Clothing_Designers__ __clothing-female__ , standing on beach in turkey, pencil eyeliner, DSLR, parted lips, pink lipstick, silver necklace with a cross on it, straw hat, Full body image,

![2024-08-12_18-02-16_6336](https://github.com/user-attachments/assets/07dcf3d9-809a-4017-bedb-677ad8bf839e)
![2024-08-12_18-03-08_8279](https://github.com/user-attachments/assets/42200c5a-429d-432e-9cbc-1a77b007d769)
![2024-08-12_18-03-59_9891](https://github.com/user-attachments/assets/2200e4c2-092e-47ac-9e38-8fdb7d7dfcf9)
![2024-08-12_18-04-51_3980](https://github.com/user-attachments/assets/943c69e1-3a95-4090-adbe-cf32717abe33)



so this means my lora is over setting the image size for full body?

**Reply by poor7** [reply]
You're right, because you're probably using a very long and detailed description in your wildcards, the rest of the prompt doesn't matter anymore, because it's just drowning in prompts (tokens). 

You could try reducing the number of prompts in your wildcards, or increasing the weight of the most significant prompts. Adding a shoe description might also help.

Try swapping it around by putting "Full body image" at the beginning, which should make it stronger, and "jessicaL" at the end.

Are you sure this is LoRA? If yes, then try to reduce the weight LoRA in LoRAs settings

**Reply by slipperyjankinz** [reply]
Do you have any styles checked?
![image](https://github.com/user-attachments/assets/2c17f650-28de-4072-9e09-5c099400842f)

This one in particular (Fooocus V2) often greatly lowers the adherence to my prompt.  In addition to removing that, try keeping the phrase "head and feet" near the beginning of your prompt.

**Comment by Rogala** [comment]
[Try this LoRA but initially it should be checked and adjusted to your character, its weight can vary from -2 to 2, and sometimes -1, 0, 1 only give normal results but in any ratio of the sides of the picture.](https://civitai.com/models/213307/envy-zoom-slider-xl-01?modelVersionId=240274)

**Comment by lgwjames** [comment]
I have been advised this is more that my lora was not trained on full images and also foocus somehow is letting the lora over write my prompts. I moved to Sd this week to test and the same thing

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
